lib/experimental/Widgets/Content/IconTextsList/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { ComponentProps, forwardRef } from "react"
+import { IconText } from "../IconText"
+
+interface IconTextsListProps {
+  list: ComponentProps<typeof IconText>[]
+}
+
+export const IconTextsList = forwardRef<HTMLDivElement, IconTextsListProps>(
+  ({ list }, ref) => {
+    return (
+      <div ref={ref} className="mt-2 flex flex-col gap-2">
+        {list.map((item) => (
+          <IconText key={item.texts[0]} icon={item.icon} texts={item.texts} />
+        ))}
+      </div>
+    )
+  }
+)
```

Comment:
How common is it to have a list of icon text components together? Is it worth having a dedicated component for it?

Reply:
The only usecase I've seen is a list (status widget)